### ISSUE TITLE
Search not working

### ISSUE THREAD

**Issue by moulibrota-das** [PrathmeshSadake/unsplashy#3]
**The Bug**
The search feature is not working. Its just reloading the same page again.

**Steps to reproduce**

1. Go to the website
2. Search any picture (ex: cars) in the search bar

**Expected Behavior**
On searching with a keyword the website should render and show particular images associated with that keyword.

**Comment by PrathmeshSadake** [comment]
Kindly start working on this issue.

**Comment by moulibrota-das** [comment]
There is an API authentication error in my local build. Do I have to get my own API key? @PrathmeshSadake

**Comment by PrathmeshSadake** [comment]
Yes, you can get your API key at https://unsplash.com/developers
Add your API keys in the .env file. Check for names in the src folder and use the same names.

**Comment by Aditya3403** [comment]
Sir I would like to work on this issue. kindly assign this issue to me.

**Comment by mrudulkolambe** [comment]
Sir I would like to work on the issue
![image](https://user-images.githubusercontent.com/83169198/157078217-c77507ed-2ec7-4f3d-8865-6218c5e02b7e.png)

**Comment by moulibrota-das** [comment]
@nawed2611 can you tell me why my pull request regarding this issue which has been submitted way earlier is not considered and has not even been provided any feedback?

![image](https://user-images.githubusercontent.com/88196189/157687918-8f7d079c-dc10-4867-aaad-cddd12101757.png)

**Comment by nawed2611** [comment]
Hey @moulibrota-das, don't worry I was occupied with some work. I'll be reviewing all the PRs carefully so it took time.

**Comment by nawed2611** [comment]
Please check the comment made on PR #11
